net/gatling/files/gatling.init

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh /etc/rc.common
+
+# shellcheck shell=ash
+
+# Just looks for changes in the config-file and applies them with a
+# one-time-run.
+
+USE_PROCD=1
+# PROCD_DEBUG=1
+
+# busybox doesn't include the rev-command. Thus implementing it by ourself.
+rev() {
+    while read -r var; do
+        # function taken from https://stackoverflow.com/a/34668251
+        rev=""
+        i=1
+
+        while [ "$i" -le "${#var}" ]; do
+            rev="$(echo "$var" | awk -v i="$i" '{print(substr($0,i,1))}')$rev"
+            : $((i += 1))
+        done
+
+        echo "$rev"
+    done
+}
+
+# taken from /etc/init.d/uhttpd
+append_arg() {
+	local cfg="$1"
+	local var="$2"
+	local opt="$3"
+	local def="$4"
+	local val
+
+	config_get val "$cfg" "$var"
+	[ -n "$val" -o -n "$def" ] && procd_append_param command "$opt" "${val:-$def}"
+}
+
+service_triggers() {
+    procd_add_reload_trigger "gatling"
+}
+
+start_instance() {
+
+    local cfg="$1"
+	local listen http ftp_server
+	local enabled
+
+	config_get_bool enabled "$cfg" 'enabled' 1
```

Comment:
This file contains tabs and spaces as indentation. Please use tabs only (that is what I see is used in other scripts).

Reply:
Adjusted it to use tabs only in the most recent version of this PR.